x-pack/legacy/plugins/siem/server/lib/kpi_hosts/elasticsearch_adapter.ts

```diff
@@ -88,7 +88,21 @@ export class ElasticsearchKpiHostsAdapter implements KpiHostsAdapter {
       'responses.2.aggregations.unique_destination_ips_histogram.buckets',
       response
     );
+
+    const inspect = {
+      dsl: [
+        inspectStringifyObject(hostsQuery[1]),
```

Comment:
Because of `msearch` api, 
the data for `hostQuery` here is in this format:
```
[
{
      index: defaultIndex,
      allowNoIndices: true,
      ignoreUnavailable: true,
    },
    {
      aggregations: {
        hosts: {
          cardinality: {
            field: 'host.name',
          },
        },
        hosts_histogram: {
          auto_date_histogram: {
            field: '@timestamp',
            buckets: '6',
          },
          aggs: {
            count: {
              cardinality: {
                field: 'host.name',
              },
            },
          },
        },
      },
      query: {
        bool: {
          filter,
        },
      },
      size: 0,
      track_total_hits: false,
    },
]
```
Now we are only showing the query body,
would we want to merge them to the same object like other queries? 
e.g.
```
{
  "allowNoIndices": true,
  "index": [
    "auditbeat-*",
    "filebeat-*",
    "packetbeat-*",
    "winlogbeat-*"
  ],
  "ignoreUnavailable": true,
  "body": {
    "aggregations": {
      "user_count": {
        "cardinality": {
          "field": "user.name"
        }
      },
      "group_by_users": {
        "terms": {
          "size": 11,
          "field": "user.name",
          "order": [
            {
              "successes.doc_count": "desc"
            },
            {
              "failures.doc_count": "desc"
            }
          ]
        },
        "aggs": {
          "failures": {
            "filter": {
              "term": {
                "event.type": "authentication_failure"
              }
            },
            "aggs": {
              "lastFailure": {
                "top_hits": {
                  "size": 1,
                  "_source": [],
                  "sort": [
                    {
                      "@timestamp": {
                        "order": "desc"
                      }
                    }
                  ]
                }
              }
            }
          },
          "successes": {
            "filter": {
              "term": {
                "event.type": "authentication_success"
              }
            },
            "aggs": {
              "lastSuccess": {
                "top_hits": {
                  "size": 1,
                  "_source": [],
                  "sort": [
                    {
                      "@timestamp": {
                        "order": "desc"
                      }
                    }
                  ]
                }
              }
            }
          },
          "authentication": {
            "top_hits": {
              "size": 1,
              "_source": [],
              "sort": [
                {
                  "@timestamp": {
                    "order": "desc"
                  }
                }
              ]
            }
          }
        }
      }
    },
    "query": {
      "bool": {
        "filter": [
          {
            "term": {
              "event.category": "authentication"
            }
          },
          {
            "range": {
              "@timestamp": {
                "gte": 1562001357755,
                "lte": 1562087757756
              }
            }
          }
        ]
      }
    }
  },
  "size": 0,
  "track_total_hits": false
}
```

Reply:
fixed in e3eb02a